code/modules/mob/mob.dm

```diff
@@ -107,6 +109,17 @@
 	. = ..()
 	tag = "mob_[next_mob_id++]"
 
+/// Assigns a (c)key to this mob.
+/mob/proc/PossessByPlayer(ckey)
```

Comment:
is this for the future? or for further safety

Reply:
Centralizing proc where clients are pushed around is good.